docs/application/dotnet/guides/nui/view.md

```diff
@@ -1,75 +1,133 @@
 # View
 
-The `View` class is the base class for all views. With this class you can manage the background color and images for UI components.
+[View](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html) is the fundamental concept for the all components such as `Button`, `Image`, `Text`, and so on.
+`View` provides [Properties](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#properties), [Methods](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#methods), and [Events](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#events) that are commonly used on every components. To show the `View`, you must add `View` on the [Window](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.Window.html) as a child.
 
-In this tutorial, the following subjects are covered:
-
-[View events](#1)<br>
-[Setting the Background Color](#2)<br>
-[Setting the Background Image](#3)<br>
 
 <a name="1"></a>
-## View events
+## View Transforms
 
-The following table lists the basic event provided by the `View` class:
+`position`, `orientation`, and `scale` of `View` are known as transforms.
+`View` supports hierarchical structure. Therefore, `position`, `orientation`, and `scale` of every `View` are affected by their parents `Views`.
 
-**Table: View events**
+`Position` of each `View` is defined as the distance between the position of [ParentOrigin](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_ParentOrigin) and *TopLeft* position of the `View`. The `ParentOrigin` is a property defines origin of its parent. For example, if a `View` has properties `Position` (5, 3) and `ParentOrigin` is *Center*, then the `View` is located (5, 3) away from *Center* position of its parent `View`.
```

Comment:
1. As `position` is in the code tag, please change `Position` to `position`.
    In line number 10, we use `position`. 

2. Please remove "the" before `ParentOrigin`.

Reply:
Thanks. Done!